### PR TITLE
Add person student teacher classes

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, I made the following

- [ ] Implement classes and objects in Ruby.
- [ ] Implement encapsulation and inheritance with Ruby.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'person'
+
+class Student < Person
+  def initialize(classroom)
```
Comment (mikethreels) [inline student.rb]: - [x] the initialize method should also take the arguments `age, name = 'Unknown', parent_permission: true` like in the student class

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'person'
+
+class Teacher < Person
+  def initialize(specialization)
```
Comment (mikethreels) [inline teacher.rb]: - [x] the initialize method should also take the arguments `age, name = 'Unknown', parent_permission: true` like in the student class

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kero, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 2 above)
Comment (keroloussamy) [reply teacher.rb]: Done

File: student.rb (hunk 1 above)
Comment (keroloussamy) [reply student.rb]: Done

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @keroloussamy, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**